src/Axiom/Set.agda

```diff
@@ -4,7 +4,7 @@ open import Agda.Primitive renaming (Set to Type)
 
 module Axiom.Set where
 
-open import Prelude hiding (filter; map)
```

Comment:
`Prelude` no longer imports `Data.List.filter`. Instead, we import the kind of filter we need where we need it, and we name it `filter`.